### DISCUSSION TITLE
Use frontend or backend?

### DISCUSSION THREAD

**Discussion by reatailret** [wintercms/winter#1025] [Q&A] [answered]
Hello! For CRM like internal system, with clients, many roles,permissions, relations,user accounts and marketplace like functions (prices, import, exports), many tables, filters , etc. should i use frontend with components or backend?

**Comment by mjauvin** [comment, marked as answer]
I would probably use backend.

**Reply by bennothommo** [reply]
This is covered in the Configuration docs if anyone is interested:
https://wintercms.com/docs/v1.2/docs/setup/configuration#backend-only-mode

**Comment by LukeTowers** [comment]
Note that you can remove the cms module entirely and set the backendUri to `''` which will turn your Winter install into a fully backend-focused web app. It's what I usually do for these sorts of projects.
